### ISSUE TITLE
2.5D

### ISSUE THREAD

**Issue by red0bear** [AlmasB/FXGL#1255]
hello there , 

i was expecting we had https://en.wikipedia.org/wiki/2.5D environments ... but i dont know if you have plans to add it .

**Comment by chengenzhao** [comment]
Is this effect you expect?

https://twitter.com/WhiteWoodCity/status/1638528428440944640

We use GameSubScene to implement this effect
the original GameScene is used for x and z axis in which we display shadows of the unit
and add a new GameSubScene(x and y axis) in which we display the real characters like 2D games
and then use binding to associate two different entities(one in the xz coordinates, another in the xy cooridnates)

For GameSubScene check the sample code here

https://github.com/AlmasB/FXGL/commit/4a70a035bddde3de54690052dad2820f925cbe3e

**Comment by red0bear** [comment]
hello, 

Nice example ... that's not exact what am looking for. Am looking more like command conquer 2.5D map just to play around if possible.

**Comment by AlmasB** [comment]
Hi, the plans are there, particularly [this one](https://github.com/AlmasB/FXGL/issues/1151) as Tiled supports isometric tiles. Though it needs a good chunk of development time to make it work out of the box.

**Comment by red0bear** [comment]
Oh ok sir. Just try some stuff here. i see your examples are more like 2d and some 3d experimental thing. But thank you to answer.

**Comment by AlmasB** [comment]
Whilst the engine does not yet understand the 2.5 mode yet, isometric support for .tmx is now ready. Here's a screenshot:

![Screenshot 2023-03-28 20 38 03](https://user-images.githubusercontent.com/3594742/228382611-279deed0-861b-426d-b841-e1a588000d61.png)

**Comment by red0bear** [comment]
yes , i see that stuff at your channel ... is a steam game . i was thinking use some svg and other stuff and make draws on inkscape just to test somethings on future. look at this ... http://freeciv.org/
